gatorgrouper/models.py

```diff
@@ -0,0 +1,71 @@
+""" This is undocumented """
```

Comment:
@aubreypc @barrezuetai  There are several `""" This is undocumented """` that I added to pass travis and `pylint`. Whoever has more understanding of these `django` files should change these into something more appropriate. This is not the only file with these `docstrings`.